server/task_check_executor_statement_advisor.go

```diff
@@ -52,6 +52,8 @@ func (exec *TaskCheckStatementAdvisorExecutor) Run(ctx context.Context, server *
 			Logger:    exec.l,
 			Charset:   payload.Charset,
 			Collation: payload.Collation,
+			Level:     payload.Level,
```

Comment:
curious why we create a separate Level field instead of putting it into the Payload.